.travis.yml

```diff
@@ -27,7 +27,7 @@ install:
 
       export PATH=/Users/travis/miniconda3/bin:$PATH
 
-      conda update --yes conda
+      # conda update --yes conda
```

Comment:
We need to check the logs as a newer version that fix the bugs was released right after the buggy one.  If we can be with the latest we should not comment this. (At least now we know it was a conda bug.)